test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+flake8
```

Comment:
These are the testing dependencies that we don't need to install when we're running the production app.